src/server/gc_worker/gc_worker.rs

```diff
@@ -364,15 +364,16 @@ where
         };
         let mut gc_info = GcInfo::default();
         let mut next_gc_key = keys.next();
+        let mut handled_keys = 0;
         while let Some(ref key) = next_gc_key {
-            GC_COMPACTION_FILTER_MVCC_DELETION_HANDLED.inc();
             if let Err(e) = self.gc_key(safe_point, key, &mut gc_info, &mut txn, &mut reader) {
                 error!(?e; "GC meets failure"; "key" => %key,);
                 // Switch to the next key if meets failure.
                 gc_info.is_completed = true;
             }
 
             if gc_info.is_completed {
+                handled_keys += 1;
```

Comment:
Why not `inc` directly here?

Reply:
`GC_COMPACTION_FILTER_MVCC_DELETION_HANDLED` is only for mvcc deletions come from compaction filter.